plugins/builds/triggers/helpers.js

```diff
@@ -1032,6 +1033,48 @@ function buildsToRestartFilter(joinPipeline, groupEventBuilds, currentEvent, cur
         .filter(build => build !== null);
 }
 
+/**
+ * Get subsequent job names which the root is the start from node
+ * @param   {Array}   [workflowGraph]         Array of graph vertices
+ * @param   {Array}   [workflowGraph.nodes]   Array of graph vertices
+ * @param   {Array}   [workflowGraph.edges]   Array of graph edges
+ * @param   {String}  [startNode]             Starting/trigger node
+ * @returns {Array<String>}                   subsequent job names
+ */
+function subsequentJobFilter(workflowGraph, startNode) {
+    const { nodes, edges } = workflowGraph;
+
+    if (!startNode || !nodes.length) {
+        return [];
+    }
+
+    let start = startNode;
+
+    // startNode can be a PR job in PR events, so trim PR prefix from node name
+    if (startNode.match(/^PR-[0-9]+:/)) {
+        start = startNode.split(':')[1];
+    }
```

Comment:
You can use these functions.

https://github.com/screwdriver-cd/screwdriver/blob/919680ba9b286704f211adc6caff347e028543c5/plugins/builds/triggers/helpers.js#L362-L364

https://github.com/screwdriver-cd/screwdriver/blob/919680ba9b286704f211adc6caff347e028543c5/plugins/builds/triggers/helpers.js#L371-L377

Reply:
I fixed it!